cockroachcloud/free-faqs.md

```diff
@@ -100,3 +100,5 @@ In the meantime, you can run a [`SELECT`](../v20.2/select.html) statement using
 ~~~
 $ cockroach sql -e 'SELECT * FROM test_database.table1' --format=csv --url='postgres://username:password@free-tier...' > users.txt
 ~~~
+
+For an example on how to use this output to migrate to a paid CockroachCloud cluster, see [Migrate from a CockroachCloud Free (beta) to CockroachCloud Cluster](migrate-from-free-to-dedicated.html).
```

Comment:
your call but if you delete the "beta" from this you never have to update it again